src/main/kotlin/graphics/scenery/primitives/ParticleGlyphs.kt

```diff
@@ -56,8 +61,9 @@ class ParticleGlyphs @JvmOverloads constructor(var positions: List<Vector3f>, va
 
     override fun createGeometry(): Geometry {
         return object: DefaultGeometry(this) {
-            override var vertices: FloatBuffer = BufferUtils.allocateFloat(positions.size * 3)
-            override var normals: FloatBuffer = BufferUtils.allocateFloat(properties.size * 3)
+            override var vertices: FloatBuffer = BufferUtils.allocateFloat(positions.limit())
```

Comment:
That should not be `positions.limit()`, but `positions.remaining()`

Reply:
And why is that? remaining is the number between the current pointer and the limit. Shouldn't the pointer alsways be on the first position?